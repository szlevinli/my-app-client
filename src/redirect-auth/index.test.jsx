import React from 'react';
import {shallow} from 'enzyme';
import RedirectAuth from './index';

it('<RedirectAuth /> should render correctly', () => {
  const wrapper = shallow(<RedirectAuth></RedirectAuth>);
  expect(wrapper).toMatchSnapshot();
});
