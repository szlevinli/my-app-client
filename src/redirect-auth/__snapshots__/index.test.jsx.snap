// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RedirectAuth /> should render correctly 1`] = `
<BrowserRouter>
  <div>
    <AuthButton />
    <ul>
      <li>
        <Link
          to="/public"
        >
          Public Page
        </Link>
      </li>
      <li>
        <Link
          to="/protected"
        >
          Protected Page
        </Link>
      </li>
    </ul>
    <Switch>
      <Route
        path="/public"
      >
        <PublicPage />
      </Route>
      <Route
        path="/login"
      >
        <LoginPage />
      </Route>
      <PrivateRoute
        path="/protected"
      >
        <ProtectedPage />
      </PrivateRoute>
    </Switch>
  </div>
</BrowserRouter>
`;
